### DISCUSSION TITLE
No Alerts in Dashboard

### DISCUSSION THREAD

**Discussion by anitatata** [Security-Onion-Solutions/securityonion#12274] [2.4] [answered]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

cloud

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

20

### Storage for /

100

### Storage for /nsm

200

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello, 
Previously I installed version 2.4.30 but there is no Alerts in Alerts tab, so I try to installed Security Onion 2.4.40 but Alerts tab still did not showing anything.

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/0474162d-f79c-4e01-91ea-f9c957a5cb45)

Below elastic-agent status
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/ba3c0a5e-77f0-4fce-ae4d-eb3b978aaae6)

Do we need to configure anything to make Alerts tab have data?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by 0nc3B1tt3n** [comment]
You can run a watch 'so-redis-count' to see if there is a backlog of logs. If there is a backlog of logs run a so-logstash-restart. Then run the first command again to see if logs begin ingesting.

**Reply by anitatata** [reply]
Thanks for replying,

Below the capture of so-redis-count
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/846bb5ca-b5dc-4e28-afbd-bbe4c740555a)


Thanks

**Comment by sandraimmaculate** [comment]
Even I'm also facing same issues when i installed security onion 2.3 the alerts been displayed in dashboard soon after the installation but in 2.4 its not happening IDK why my alert dashboard is empty. Is this because we add another NIC card for monitoring if this is a reason how to make it work 
even I'm also getting same output for the command so-redis-count the output is 0 
help us to solve this issues

**Reply by sandraimmaculate** [reply]
yea! now i got the alerts in dashboard u can perform the command **so-test** and then check in alert dashboard your getting alert are not. if still not enable suricata and zeek in administration > configuration.

and execute the command: 
(nids testing)
1. curl testmynids.org/uid/index.html for 
2. curl -sSL https://raw.githubusercontent.com/0xtf/testmynids.org/master/tmNIDS -o /tmp/tmNIDS && chmod +x /tmp/tmNIDS &&   /tmp/tmNIDS

**Reply by anitatata** [reply]
Thanks for your suggestion.

I try the command so-test
<img width="718" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/221541af-d2e8-444b-9121-3f97deda8524">

And also make sure suricata and zeek enabled
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/06654650-e7b5-411e-a367-80d9a953bc77)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/9ff6b21d-53d6-49b9-91c2-09dc29becdfe)

But still the alerts tab did not show anything.
Is there other configuration that I missed?

Thanks

**Comment by sandraimmaculate** [comment]
did you do NIDS test is that showing output.
and did you enabled idstools which is in administration > configuration

**Reply by anitatata** [reply]
Yes, I can curl to the NIDS test
1. curl testmynids.org/uid/index.html

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/0af92abf-160a-42d6-a98c-76f148cdde82)

2. curl -sSL https://raw.githubusercontent.com/0xtf/testmynids.org/master/tmNIDS -o /tmp/tmNIDS && chmod +x /tmp/tmNIDS && /tmp/tmNIDS (but I not using /tmp but my home directory)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/10ade1b1-4b8d-43e2-b193-7c8eb5b78f71)
 and also already enabled idstools
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/f01d5336-decf-4a90-89b0-bb83865ebaae)

**Reply by anitatata** [reply]
Do you use suricata or zeek in your mdengine?
currently i'm using zeek, should I change to suricata? and follow this link?
https://docs.securityonion.net/en/2.4/suricata.html

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/88fa3331-9315-4003-87e3-5b77452dd58d)

**Comment by reyesj2** [comment]
Are you deploying via one of our cloud images?

**Reply by anitatata** [reply]
Yes I'm deploying via this link
https://download.securityonion.net/file/securityonion/securityonion-2.4.40-20240116.iso

**Comment by anitatata** [comment]
Any suggestion guys? My security onion still has no alerts 
Please Help

**Reply by cm-ops** [reply]
What doe `ls - la /nsm/suricata` return?

**Reply by zazman49** [reply]
guys me too Alerts tab show nothing, I tried all steps and no result
??

**Reply by zazman49** [reply]
<img width="362" alt="Screenshot 2024-02-18 012030" src="https://github.com/Security-Onion-Solutions/securityonion/assets/156570099/b5b55617-7507-465d-8115-ffa4b26d705e">

**Reply by anitatata** [reply]
Below is the capture 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/bb837935-019f-4c5f-8cbd-32d90bdb36b1)

**Reply by cm-ops** [reply]
Those eve logs look empty.

**Reply by anitatata** [reply]
Yes the log is empty, is there anything I can do to make the alerts tab showing?

**Reply by cm-ops** [reply]
Do you see a Suricata index if you run` sudo so-elasticsearch-indices-list`?

**Reply by anitatata** [reply]
Below is the output when I run 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/e3c8f0e7-9efc-4662-9710-c6d7c80a7a31)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/c20cc4bc-b5a1-4b99-aee2-dab685eff530)

**Comment by zazman49** [comment, marked as answer]
solved , by moving the endpoints from Elastic fleet policies into so-grid and it worked

**Reply by anitatata** [reply]
Hello can you tell more about how to do it?

**Reply by zazman49** [reply]
first step click on elastic fleet --> choose agents endpoint  ---> select the three dots on the right --> move them to so-grid policy or the policy you chosse
![1](https://github.com/Security-Onion-Solutions/securityonion/assets/156570099/3dcff5ba-2fe0-4360-91e8-55abd5df8dc2)

![22](https://github.com/Security-Onion-Solutions/securityonion/assets/156570099/287c2d38-badf-490a-a76b-f6a3485673f0)
![2](https://github.com/Security-Onion-Solutions/securityonion/assets/156570099/c0871c34-0df9-41e0-bd9e-d24c61088696)

![4](https://github.com/Security-Onion-Solutions/securityonion/assets/156570099/aff02ffb-2200-4078-a7cd-5b486d3d71f9)
